samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/model/inline_object2.dart

```diff
@@ -27,10 +27,10 @@ abstract class InlineObject2 implements Built<InlineObject2, InlineObject2Builde
 class InlineObject2EnumFormStringArray extends EnumClass {
 
   /// Form parameter enum test (string array)
-  @BuiltValueEnumConst(wireName: '>')
+  @BuiltValueEnumConst(wireName: r'>')
   static const InlineObject2EnumFormStringArray greaterThan = _$inlineObject2EnumFormStringArray_greaterThan;
   /// Form parameter enum test (string array)
-  @BuiltValueEnumConst(wireName: '$')
+  @BuiltValueEnumConst(wireName: r'\$')
```

Comment:
Found a mismatch between dart-dio and dart. a few files below you will find:

```
static const dollar = EnumArraysJustSymbolEnum._(r'$');
```

You can see the `$` isn't escaped with `\` but it is here, both are raw strings. I suspect this is breaking compatibility between them?

Reply:
See the comment here, this is specific to `built_value`s Dart generator: https://github.com/OpenAPITools/openapi-generator/pull/8149/files#diff-f325b1ca8b03ba6f1ee25144eec1079de2cc2d853d6237eac40f157663bda2acR120

Reply:
ah thanks for that, missed comment. Well, it's fine then I expressed my opinion on use of built_value in another PR.